tests/sentry/tasks/test_unmerge.py

```diff
@@ -33,6 +34,29 @@
 index = _make_index_backend(redis.clusters.get('default').get_local_client(0))
 
 
+def test_get_fingerprint():
+    assert get_fingerprint(
+        Event(
+            data={
+                'sentry.interfaces.Message': {
+                    'message': 'Hello world',
+                },
+            },
+        )
+    ) == hashlib.md5('Hello world').hexdigest()
```

Comment:
This seems like it might know too much about how fingerprints are generated. I could instead compare that these two events are not equal if we'd prefer